app/script/telemetry/calling/CallTelemetry.js

```diff
@@ -101,109 +109,68 @@ z.telemetry.calling.CallTelemetry = class CallTelemetry {
 
   /**
    * Reports call events for call tracking to Localytics.
-   * @param {z.tracking.EventName} event_name - String for call event
-   * @param {z.calling.entities.CallEntity} call_et - Call entity
+   * @param {z.tracking.EventName} eventName - String for call event
+   * @param {z.calling.entities.CallEntity} callEntity - Call entity
    * @param {Object} [attributes={}] - Attributes for the event
    * @returns {undefined} No return value
    */
-  track_event(event_name, call_et, attributes = {}) {
-    if (call_et) {
-      const {conversationEntity, isGroup, maxNumberOfParticipants} = call_et;
+  track_event(eventName, callEntity, attributes = {}) {
+    if (callEntity) {
+      const {conversationEntity, isGroup, direction} = callEntity;
+
+      const videoTypes = [z.media.MediaType.VIDEO, z.media.MediaType.AUDIO_VIDEO];
 
       attributes = Object.assign(
         {
           conversation_participants: conversationEntity.getNumberOfParticipants(),
-          conversation_participants_in_call: maxNumberOfParticipants ? maxNumberOfParticipants : undefined,
+          conversation_participants_in_call: this.maxNumberOfParticipants ? this.maxNumberOfParticipants : undefined,
```

Comment:
This was renamed to contain a `max` in the name

Reply:
nice catch